src/proxy/inbound_passthrough.rs

```diff
@@ -55,7 +55,12 @@ impl InboundPassthrough {
                         }
                     });
                 }
-                Err(e) => error!("Failed TCP handshake {}", e),
+                Err(e) => {
+                    if e.get_ref().unwrap().to_string().eq("shutdown") {
```

Comment:
This should be `.contains()` instead of `.eq()`, like in the other places.

Reply:
this is my mistake